CHANGELOG.md

```diff
@@ -25,11 +25,20 @@ Here is a template for new release sections
 - `E-Land_Requirements.rst`: Official E-Land requirement list as well as progress on functional and non-functional requirements (#590)
 - Add pytests for `E4.detect_excessive_excess_generation_in_bus()` (#591)
 - Add pypi release to release protocol and update/simplify protocol (#601)
+- Remove REPORT_PATH constant (#607)
+- Add report assets and example simulation to package_data in `setup.py` (#607)
+- Add a util function to copy report asset to simulation output folder when user generates the report (#607)
+- Add entrypoints for `mvs_tool` and `mvs_report` in ´setup.py´ (#607)
```

Comment:
Leave hint that this allows a simpler terminal command (the other one is still possible, isnt it?)

Reply:
`python mvs_report.py` not anymore. `python mvs_tool.py` only if you are located at the root of the repo, but it does exactly the same as `mvs_tool`, so why would you like to type more in the terminal?

Reply:
Because it is still stored in my history xD

Reply:
Done in 1f71fb5e4f004777ae3ac5799d8e1c0341159728